lib/active_merchant/billing/gateways/redsys.rb

```diff
@@ -266,9 +266,13 @@ def refund(money, authorization, options = {})
       end
 
       def verify(creditcard, options = {})
-        MultiResponse.run(:use_first_response) do |r|
-          r.process { authorize(100, creditcard, options) }
-          r.process(:ignore_result) { void(r.authorization, options) }
+        if options[:sca_exemption_behavior_override] = 'endpoint_and_ntid'
```

Comment:
Does this value get passed to `options` when included as a gateway specific field?

Reply:
Looks like the answer is yes, if we add the `on: :all` property.
We will also need to do this for `sca_exemption`